client/homescreen/activity-panel/panels.js

```diff
@@ -56,6 +56,7 @@ export function getAllPanels( {
 				title: __( 'Stock', 'woocommerce-admin' ),
 			},
 		publishedProductCount > 0 &&
+			countUnapprovedReviews > 0 &&
```

Comment:
Just noting that I would expect this name to be `unapprovedReviewsCount` but I can see that's pre-existing so not necessary to change here unless you're up for it.

Reply:
@joshuatf I like your suggestion, I'm going to merge this PR and address that change, renaming [countLowStockProducts](https://github.com/woocommerce/woocommerce-admin/blob/main/client/homescreen/activity-panel/panels.js#L14) and [countUnreadOrders](https://github.com/woocommerce/woocommerce-admin/blob/main/client/homescreen/activity-panel/panels.js#L16) as well.